vendor/github.com/TykTechnologies/tykcommon/api_definitions.go

```diff
@@ -315,7 +315,7 @@ type APIDefinition struct {
 		StripListenPath             bool                          `bson:"strip_listen_path" json:"strip_listen_path"`
 		EnableLoadBalancing         bool                          `bson:"enable_load_balancing" json:"enable_load_balancing"`
 		Targets                     []string                      `bson:"target_list" json:"target_list"`
-		StructuredTargetList        *HostList                     `bson:"-" json:"-"`
+		StructuredTargetList        HostList                      `bson:"-" json:"-"`
```

Comment:
This is my fault though, because I too forgot to update this file when updating tykcommon.